src/trans/internal/suleg_mod.F90

```diff
@@ -12,6 +12,9 @@ MODULE SULEG_MOD
 #ifdef __NEC__
 #define SIZEOF(x) STORAGE_SIZE(x)/KIND(x)
 #endif
+#ifdef NAG
```

Comment:
Looking at paragraph 4.5.4 in 
https://support.nag.com/nagware/np/r72_doc/manual/compiler_4_5.html#MACRO
suggests that this should perhaps be NAGFOR instead.
Thoughts?

Reply:
Yes, thanks, I didn't realize nag set that.  I have updated the branch.